vmicore/src/lib/vmi/Breakpoint.cpp

```diff
@@ -34,7 +35,9 @@ namespace VmiCore
 
     BpResponse Breakpoint::callback(IInterruptEvent& event)
     {
-        if (!global && event.getCr3() != dtb)
+        // Sanitize cr3 in order to get rid of potential architecture specific values that do not belong to the actual
```

Comment:
Architecture-specific stuff and also code reuse from the plugin. Should be hidden within the API.

Reply:
Hm, right now the project is not architecture agnostic at all. But I guess, we could go ahead and "future proof" this code here, as you have planned to add ARM support somewhere down the road. :thinking:

Reply:
Removed, because sanitization is not required.